_posts/2019-08-21-cockpit-201.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Cockpit 201
+author: Gundersanne
+date: 2019-08-21
+tags: cockpit linux
+slug: cockpit-201
+category: release
+summary: Cockpit with Machines, services, and system improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.
+
+
+### Machines: VM creation changes
+
+Creating a VM by importing an existing disk has been moved to a separate dialog.
+
+![Machines creating a VM by importing an existing disk](/images/machines-create-importing-disk-separate-dialog.png){:.screenshot}
+
+The Operating System input field is now mandatory.
```

Comment:
@KKoukiou wrote:

>but pls explain that this is now mandatory because it allows the guest device assignment to be optimized leading to better performance

Reply:
Oh good spot, i'll add that